jupyterlab_gitsync/src/service/file.ts

```diff
@@ -91,7 +101,7 @@ export class File implements IFile {
       const contents = await fs.get(this.path);
       this.resolver.addVersion(contents.content, 'remote');
     } catch (error) {
-      console.log(error);
+      console.warn(error);
```

Comment:
UI error needs to be exposed to users. Console.warn is not a good way to expose, since end users normally don't open developer console.

Reply:
I'm thinking of displaying the errors somewhere on the UI (maybe clicking the status indicator will show a panel with a log of all service operations, ie. save/sync/etc.), but this will most likely be included in a separate PR.